src/api/status/src/views/status.hbs

```diff
@@ -45,157 +45,11 @@
-                    22 DEC 7:20 PM
-                  </p>
-                </div>
-              </div>
-              <div class="timeline-block mb-3">
-                <span class="timeline-step">
-                  <i class="material-icons text-danger text-gradient">code</i>
-                </span>
-                <div class="timeline-content">
-                  <h6 class="text-dark text-sm font-weight-bold mb-0">New order #1832412</h6>
-                  <p class="text-secondary font-weight-bold text-xs mt-1 mb-0">21 DEC 11 PM</p>
-                </div>
-              </div>
-              <div class="timeline-block mb-3">
-                <span class="timeline-step">
-                  <i class="material-icons text-info text-gradient">shopping_cart</i>
-                </span>
-                <div class="timeline-content">
-                  <h6 class="text-dark text-sm font-weight-bold mb-0">
-                    Server payments for April
-                  </h6>
-                  <p class="text-secondary font-weight-bold text-xs mt-1 mb-0">
-                    21 DEC 9:34 PM
-                  </p>
-                </div>
-              </div>
-              <div class="timeline-block mb-3">
-                <span class="timeline-step">
-                  <i class="material-icons text-warning text-gradient">credit_card</i>
-                </span>
-                <div class="timeline-content">
-                  <h6 class="text-dark text-sm font-weight-bold mb-0">
-                    New card added for order #4395133
-                  </h6>
-                  <p class="text-secondary font-weight-bold text-xs mt-1 mb-0">
-                    20 DEC 2:20 AM
-                  </p>
-                </div>
-              </div>
-              <div class="timeline-block mb-3">
-                <span class="timeline-step">
-                  <i class="material-icons text-primary text-gradient">key</i>
-                </span>
-                <div class="timeline-content">
-                  <h6 class="text-dark text-sm font-weight-bold mb-0">
-                    Unlock packages for development
-                  </h6>
-                  <p class="text-secondary font-weight-bold text-xs mt-1 mb-0">
-                    18 DEC 4:54 AM
-                  </p>
-                </div>
-              </div>
-              <div class="timeline-block">
-                <span class="timeline-step">
-                  <i class="material-icons text-dark text-gradient">payments</i>
-                </span>
-                <div class="timeline-content">
-                  <h6 class="text-dark text-sm font-weight-bold mb-0">New order #9583120</h6>
-                  <p class="text-secondary font-weight-bold text-xs mt-1 mb-0">17 DEC</p>
-                </div>
-              </div>
-            </div>
-          </div>
-        </div>
-      </div>
     </div>
+    
```

Comment:
In general, you want to avoid changing lines of code unrelated to your fix.  It messes with the git history for these lines, and makes it look like you last touched them, when in reality, they were really written/changed in a previous commit.  We call these "spurious white space changes" and they are to be avoided.

This isn't a big issue here, but FYI for your next fix.

Reply:
Sure, I will change it back. I think it has to do with the configuration of my VSCode environment.